docs/canarytokens/queries.md

```diff
@@ -98,10 +98,353 @@ minted on your Console. A cleaner option is to use [Search Canarytokens](#search
 
 <APIDetails :endpoint="$page.frontmatter.endpoints.fetch"/>
 
+#### Example
+
+:::: tabs :options="{ useUrlFragment: false }"
+
+::: tab "cURL"
+
+``` bash
+curl https://EXAMPLE.canary.tools/api/v1/canarytokens/fetch \
+  -d auth_token=EXAMPLE_AUTH_TOKEN \
+  -G
+```
+
+:::
+
+::: tab "Python"
+
+``` python
+import requests
+
+url = 'https://EXAMPLE.canary.tools/api/v1/canarytokens/fetch'
+
+payload = {
+  'auth_token': 'EXAMPLE_AUTH_TOKEN'
+}
+
+r = requests.get(url, params=payload)
+
+print(r.json())
+```
+
+:::
+
+::::
+
+
+::: api-response
```

Comment:
For this response, we are expecting it to be long as it doesn't paginate. Instead of pasting this large response, let's cut it down to emphasise that it'll return lots of results:

```json
{
  "result": "success",
  "tokens": [
    {
      "browser_scanner_enabled": true,
      "canarytoken": "<token_code>",
      "created": "1585947447.092035",
      "created_printable": "2020-04-03 20:57:27 (UTC)",
      "enabled": false,
      "flock_id": "flock:default",
      "hostname": "<token_hostname>",
      "key": "<token_key>",
      "kind": "http",
      "memo": "Example Web Token",
      "node_id": "<node_id>",
      "triggered_count": 0,
      "updated_id": 2,
      "url": "<token_url>"
    },
    ...
  ]
}
```